Misc/NEWS.d/next/Core and Builtins/2017-12-05-21-33-47.bpo-32225.ucKjvw.rst

```diff
@@ -0,0 +1,2 @@
+PEP 562: Add support for module __getattr__ and __dir__. Implemented by Ivan
+Levkivskyi.
```

Comment:
Escape the underscores to make this valid reST:

```rst
module ``__getattr__`` and ``__dir__``
```

Reply:
Thanks, fixed!